src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -330,13 +346,53 @@ internal IEnumerable<V> Search(string query, int minResultsForTolerantSearch = 0
                 ComputeWeightAndAddToDictionary(query, pair, searchDict);
             }
 
-            return searchDict
-                .OrderByDescending(x => x.Value)
-                .Select(x => x.Key);
+            var orderedSearchDict = searchDict.OrderByDescending(x => x.Value);
+#if DEBUG
+            //if (this.logger != null)
```

Comment:
@ColinDayOrg did you mean to leave this commented out?

Reply:
Yes. Normally I would delete code like this, but it seemed that it could possibly be useful in the future. It can be removed if that would be preferable. It can output a lot of logging data and may not need to be used a lot which is why I commented it out.

Reply:
Due to this code possibly outputting a large amount of logging data if it were uncommented, and it not needing to be used currently, I have removed this code. It can always be put back if needed.